client/components/empty-content/style.scss

```diff
@@ -0,0 +1,10 @@
+/** @format */
+
+.woocommerce-empty-content {
+	text-align: center;
+
+	.woocommerce-empty-content__action.is-primary {
+		color: $white;
```

Comment:
Why is this necessary, shouldn't the button already be white?

Reply:
nevermind, I see it's a problem when the button is a link - we should fix that in the global scss file, not per-component.